semantic/src/site/modules/progress.overrides

```diff
@@ -1,3 +1,12 @@
 /*******************************
          Site Overrides
 *******************************/
+
+.ui.progress {
+  border-radius: 0em;
+}
+
+.ui.progress .bar {
+  height: 5px;
```

Comment:
Doesn't look like the "height" property is doing anything. In the inspector it's crossed out, which means another property somewhere else is taking priority. So can probably remove from here.